custom_components/tuya_local/devices/tuya_zigbee_ultrasonic_watermeter.yaml

```diff
@@ -0,0 +1,81 @@
+name: Water meter
+products:
+  - id: zlwr0raf
+    name: Smart Water Meter Zigbee
```

Comment:
This should be linked to a specific product, not a generic description.

Reply:
Added "Tuya". This product is known by many names, none of them more specific than this. I think it has been translated from Chinese by several different companies.